cpp/test/sg/kmeans_test.cu

```diff
@@ -85,14 +86,14 @@ class KmeansTest : public ::testing::TestWithParam<KmeansInputs<T>> {
                               params.n_clusters - 1);
 
     if (score < 1.0) {
-      std::cout << "Expected: "
-                << arr2Str(d_labels_ref, 25, "d_labels_ref", handle.getStream())
-                << std::endl;
-      std::cout << "Actual: "
-                << arr2Str(d_labels, 25, "d_labels", handle.getStream())
-                << std::endl;
-
-      std::cout << "Score = " << score << std::endl;
+      std::stringstream ss;
```

Comment:
Same here. This seems like something I'd only want to see if I was seeing failures in the kmeans tests and would want to drop the level to DEBUG so I could troubleshoot.

Reply:
Done.